### DISCUSSION TITLE
Beeing able to link to a specific note

### DISCUSSION THREAD

**Discussion by DavidFreely** [phibr0/obsidian-commander#15] [Q&A]
Hi,

i like to click on a button to get to a chosen note of the vault, or to an external page (websitelink, or explorer link).

Is it possible to do so?

**Comment by circuitsnacks** [comment]
Have you tried OpenGate plugin for opening website links?

**Reply by DavidFreely** [reply]
Hi, not yet. Thank you for the hint. The most importend part tho is the button to a custom chosen note. Website links or explorer links would have been just a nice extra :)

**Comment by daxmonson** [comment]
@DavidFreehily You can do this with the plugin "hotkeys for specific files" and then binding that specific hotkey shortcut you made to that file to an icon in Commander.
